content/en/docs/reference/kubectl/cheatsheet.md

```diff
@@ -207,7 +207,7 @@ kubectl rollout history deployment/frontend                      # Check the his
 kubectl rollout undo deployment/frontend                         # Rollback to the previous deployment
 kubectl rollout undo deployment/frontend --to-revision=2         # Rollback to a specific revision
 kubectl rollout status -w deployment/frontend                    # Watch rolling update status of "frontend" deployment until completion
-
+kubectl rollout restart deployment/frontend                      # Roll-out restart the "frontend" deployment
```

Comment:
nit: just `# Restart the "frontend" deployment`

Reply:
I wanted to make it clear that this is going to restart pods one-by-one

Reply:
nit:
I like ```Restart the "frontend" Deployment```
OR ```Rolling restart of the "frontend" Deployment```
(Noted that lowercase deployment is used in the other descriptions)